keyboards/nuphy/air75_v2/ansi/info.json

```diff
@@ -0,0 +1,271 @@
+        "rows": ["C14", "C15", "A0", "A1", "A2", "A3"]
+    },
+    "diode_direction": "COL2ROW",
+    "rgb_matrix": {
+      "driver": "ws2812",
+      "center_point": [80, 30],
+      "max_brightness": 128,
+      "val_steps": 26,
+      "speed_steps": 52,
+      "sleep": true,
+      "animations": {
+          "gradient_up_down": true,
+          "gradient_left_right": true,
+          "breathing": true,
+          "band_sat": true,
+          "band_val": true,
+          "band_pinwheel_sat": true,
+          "band_pinwheel_val": true,
+          "band_spiral_sat": true,
+          "band_spiral_val": true,
+          "cycle_all": true,
+          "cycle_left_right": true,
+          "cycle_up_down": true,
+          "rainbow_moving_chevron": true,
+          "cycle_out_in": true,
+          "cycle_out_in_dual": true,
+          "cycle_pinwheel": true,
+          "cycle_spiral": true,
+          "dual_beacon": true,
+          "rainbow_beacon": true,
+          "rainbow_pinwheels": true,
+          "raindrops": true,
+          "jellybean_raindrops": true,
+          "hue_breathing": true,
+          "hue_pendulum": true,
+          "hue_wave": true,
+          "typing_heatmap": true,
+          "digital_rain": true,
+          "solid_reactive_simple": true,
+          "solid_reactive": true,
+          "solid_reactive_wide": true,
+          "solid_reactive_multiwide": true,
+          "solid_reactive_cross": true,
+          "solid_reactive_multicross": true,
+          "solid_reactive_nexus": true,
+          "solid_reactive_multinexus": true,
+          "splash": true,
+          "multisplash": true,
+          "solid_splash": true,
+          "solid_multisplash": true
+        },
+        "layout": [
+            {"matrix": [0, 0], "x": 0, "y": 0, "flags": 4},
+            {"matrix": [0, 1], "x": 10, "y": 0, "flags": 4},
+            {"matrix": [0, 2], "x": 20, "y": 0, "flags": 4},
+            {"matrix": [0, 3], "x": 30, "y": 0, "flags": 4},
+            {"matrix": [0, 4], "x": 40, "y": 0, "flags": 4},
+            {"matrix": [0, 5], "x": 50, "y": 0, "flags": 4},
+            {"matrix": [0, 6], "x": 60, "y": 0, "flags": 4},
+            {"matrix": [0, 7], "x": 70, "y": 0, "flags": 4},
+            {"matrix": [0, 8], "x": 80, "y": 0, "flags": 4},
+            {"matrix": [0, 9], "x": 90, "y": 0, "flags": 4},
+            {"matrix": [0, 10], "x": 100, "y": 0, "flags": 4},
+            {"matrix": [0, 11], "x": 110, "y": 0, "flags": 4},
+            {"matrix": [0, 12], "x": 120, "y": 0, "flags": 4},
+            {"matrix": [2, 14], "x": 130, "y": 0, "flags": 4},
+            {"matrix": [0, 15], "x": 140, "y": 0, "flags": 4},
+            {"matrix": [0, 14], "x": 150, "y": 0, "flags": 4},
+
+            {"matrix": [1, 16], "x": 150, "y": 10, "flags": 4},
+            {"matrix": [1, 13], "x": 130, "y": 10, "flags": 4},
+            {"matrix": [1, 13], "x": 130, "y": 10, "flags": 4},
```

Comment:
Duplicate

Reply:
Hi DP19 @DP19 ，I have fixed this error, Thanks